src/broadcast/broadcastmanager.cpp

```diff
@@ -6,14 +6,15 @@
 #include "engine/sidechain/enginesidechain.h"
 #include "soundio/soundmanager.h"
 
-BroadcastManager::BroadcastManager(UserSettingsPointer pConfig,
+BroadcastManager::BroadcastManager(SettingsManagerPointer pSettingsManager,
```

Comment:
@daschuer Okay. If thinking this way, I think I should use shared pointer for `BroadcastProfile` instances.